### DISCUSSION TITLE
Add support for Wrap Text in detail list + tooltip

### DISCUSSION THREAD

**Discussion by mayabank** [microsoft-search/pnp-modern-search#1365] [Ideas]
**Version used**
4.2

**Describe the bug**
In the Search Result Web-Part, the column headings are not wrapping. Instead it shows up three dots (...) indicating to expand column to see the column to full width. Please advise how the column headings be word wrapped by default. 

![image](https://user-images.githubusercontent.com/84386411/133922160-d4f77ae4-cb28-4b28-b312-ca413e4e49ba.png)

**To Reproduce**
1. Go to : Page wherein Search Result web part is installed and search something with 20 columns on the search result. Have their width setup to 100 maximum.
2. Click on : Search to get the results and have the search results populated
4. See error: The column headings are no word wrapped.

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Additional context**

**Comment by wobba** [comment]
I assume you are using the detailed layouts control?  And we are using the UI Fabric `DetailsList`. I cannot see a setting to control header wrapping in https://docs.microsoft.com/en-us/javascript/api/react/idetailsheaderbaseprops so I suggest you reduce the number of columns or create a custom template with your own CSS/markup.

Seems we can add a workaround as suggested in https://stackoverflow.com/questions/64716971/is-there-a-way-to-make-a-detailslist-header-text-wrap-in-fluent-ui but should rather be exposed as a setting which require more work.

Tagging as enhancement, as it's not a bug.

**Comment by mayabank** [comment]
Hello @wobba,

Thank you for feedback. Would there be any setting to enable tool tip on the column heading? Could there be any detailed layout template that I can readily use?

Having the feature to word wrap column heading is highly appreciable. 

Thanks and regards,
Mayank
